app/controllers/v0/post911_gi_bill_statuses_controller.rb

```diff
@@ -78,14 +78,14 @@ def user_json(user)
         participant_id: user.participant_id,
         vet360_id: user.vet360_id,
         ssn: user.ssn,
-        birth_date: iso8601_birth_date(user)
+        birth_date: iso8601_birth_date(user.birth_date)
       }.to_json
     end
 
-    def iso8601_birth_date(user)
-      return nil unless user&.va_profile&.birth_date
+    def iso8601_birth_date(birth_date)
```

Comment:
Should we not just assume that the birth_date should be in ISO8601 format? Seems like to me that formatting is a presentation thing and we should send and expect to get ISO8601.

Reply:
This is a good idea. I'm gonna make a new Zenhub issue focused only on birth_date, and split this PR off to that. I'll put this recommendation in the issue description. I think we have a couple places that want birth_date in another format, but it should be up to them to massage it out of ISO8601 into whatever form they want